ports/atmel-samd/mpconfigport.h

```diff
@@ -181,302 +64,31 @@ typedef long mp_off_t;
 #ifdef SAMD51
 #define CIRCUITPY_MCU_FAMILY                        samd51
 #define MICROPY_PY_SYS_PLATFORM                     "MicroChip SAMD51"
-#define PORT_HEAP_SIZE                              (0x20000) // 128KiB
+// 128KiB heap
+#define PORT_HEAP_SIZE                              (0x20000)
```

Comment:
I don't think we use this at all. It should all be dynamic now.

Reply:
Yep, defined a lot of places but not used at all. I'll remove